src/commands/fix_lock_files/mod.rs

```diff
@@ -34,13 +37,19 @@ pub struct Options {
 /// dependencies slip into CI.
 ///
 /// Any workspaces containing a ".no_cargo_lock" sentinel file will be skipped.
-pub fn fix_workspace_lockfile(
-    repo: &Repository,
+pub async fn fix_workspace_lockfile(
+    repo_root: &Path,
     workspace_path: &Path,
     head_rev: String,
     base_rev: Option<String>,
+    changed_lockfiles: &HashSet<PathBuf>,
     check: bool,
-) -> anyhow::Result<()> {
+) -> anyhow::Result<(String, String, bool)> {
```

Comment:
Not a fan of the opaque `(String, String, bool)`. You should make a struct.

Reply:
I agree, I've open an issue to track this. It's a big refactor in the tests command